snaps_boot/ansible_p/setup/drp_images_create.yaml

```diff
@@ -14,5 +14,6 @@
 - hosts: localhost
 
   tasks:
-  - name: Upload Ubuntu 16.04 ISO to DRP
-    command: drpcli bootenvs uploadiso ubuntu-16.04-install
+  - name: Upload Ubuntu 18.04 ISO to DRP
+    command: drpcli bootenvs uploadiso ubuntu-18.04-install
+    ignore_errors: True
```

Comment:
Why ignore errors here?

Reply:
The recent DRP version typically already had the OS in place when this step is run, so adding ignore_errors to proceed if this is the case.

Reply:
I think it would be better to check for the image's existence either before or after. Troubleshooting issues that could arise here may be a bit difficult to track down. I think this would be ok for now as long as we have another issue to improve this routine.